graphql_compiler/tests/snapshot_tests/test_query_pagination.py

```diff
@@ -919,13 +973,55 @@ def test_parameter_value_generation_uuid(self):
         ]
         self.assertEqual(expected_parameters, list(generated_parameters))
 
+    @pytest.mark.usefixtures("snapshot_orientdb_client")
+    def test_parameter_value_generation_mssql_uuid(self):
```

Comment:
In the spirit of regression testing, can you include at least one test where you test that pagination works correctly if there is an existing filter on an uuid field?